### ISSUE TITLE
NTR: JC virus PCR in CSF positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8069]
New term request JC virus PCR in CSF positive (samant_110123155221): John Cunningham virus; human polyomavirus 2; cerebrospinal fluid; polymerase chain reaction; DNA  UMLS: 4324391

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000552
- label: Positive CSF JC-virus nucleic acid test
- parents: [Positive CSF nucleic acid pathogen test (HP:0430140)](https://hpo.jax.org/app/browse/term/HP:0430140)
- pmid: 
- def: Detection of JC-virus nucleic acid in the cerebrospinal fluid (CSF) by a method such as polymerase chain reaction.
- comment: JC-virus is also known as JC-Polyomavirus and John Cunningham virus.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8069
- synonyms:
